services/wallet/transfer/transaction.go

```diff
@@ -67,22 +78,28 @@ const (
 )
 
 type PendingTransaction struct {
-	Hash               common.Hash    `json:"hash"`
-	Timestamp          uint64         `json:"timestamp"`
-	Value              bigint.BigInt  `json:"value"`
-	From               common.Address `json:"from"`
-	To                 common.Address `json:"to"`
-	Data               string         `json:"data"`
-	Symbol             string         `json:"symbol"`
-	GasPrice           bigint.BigInt  `json:"gasPrice"`
-	GasLimit           bigint.BigInt  `json:"gasLimit"`
-	Type               PendingTrxType `json:"type"`
-	AdditionalData     string         `json:"additionalData"`
-	ChainID            uint64         `json:"network_id"`
-	MultiTransactionID int64          `json:"multi_transaction_id"`
+	Hash               common.Hash            `json:"hash"`
+	Timestamp          uint64                 `json:"timestamp"`
+	Value              bigint.BigInt          `json:"value"`
+	From               common.Address         `json:"from"`
+	To                 common.Address         `json:"to"`
+	Data               string                 `json:"data"`
+	Symbol             string                 `json:"symbol"`
+	GasPrice           bigint.BigInt          `json:"gasPrice"`
+	GasLimit           bigint.BigInt          `json:"gasLimit"`
+	Type               PendingTrxType         `json:"type"`
+	AdditionalData     string                 `json:"additionalData"`
+	ChainID            uint64                 `json:"network_id"`
+	MultiTransactionID MultiTransactionIDType `json:"multi_transaction_id"`
```

Comment:
Should we use `MultiTransactionIDType` for `MultiTransactionResult.ID` as well?

Reply:
I left it out for now because it is used in the public API; not sure if it doesn't require custom json parsing